src/metabase/pulse/render/body.clj

```diff
@@ -526,16 +527,6 @@
   (conj (repeat "bar")
         "line"))
 
-(defn- join-series
```

Comment:
`join-series` is removed altogether. Since the logic doesn't account for multiple questions each could have multiple series e.g. questions with multiple metrics or 2 dimensions (breakouts)